README.md

```diff
@@ -213,6 +216,52 @@ jobs:
     message: "${{ github.actor }} requests approval to run workflow."
 ```
 
+## Inputs
+
+### Required
+
+#### `bot-token`
+
+The Slack API bot token for your custom app with `chat:write` scope.
+
+#### `channel-id`
+
+The ID of a Slack channel your bot is in to send notifications to.
```

Comment:
The wording here is a bit awkward/confusing...if there is a reasonable way to improve this, that would be excellent

Reply:
Took another stab at it; let me know what you think!

Reply:
Much easier to understand--thanks!